NEWS.md

```diff
@@ -4,6 +4,21 @@
 
 ### Breaking changes
 
+- Updated rust-polars to 0.40.0 (#1104):
+  - `$join()` gains an argument `coalesce`.
+  - `$top_k()` and `$bottom_k()` gain three arguments `nulls_last`,
+    `maintain_order` and `multithreaded`.
+  - All `$rolling_*()` functions lose the arguments `by`, `closed` and
+    `warn_if_unsorted`. Rolling computations based on `by` must be made via the
+    corresponding `rolling_*_by()`, e.g `rolling_mean_by()` instead of
+    `rolling_mean(by =)`.
+  - `pl$scan_parquet()` and `pl$read_parquet()` gain an argument `glob` which
+    defaults to `TRUE`. Set it to `FALSE` to avoid considering `*` as a globing
+    pattern.
+  - `pl$read_ipc()` loses the argument `memory_map`.
```

Comment:
Again, I think this is incorrect. But it can be fixed later (no need to wait a large amount of compile time again for a single line of md file)

Reply:
I can replace by "When reading from an Arrow file with `pl$scan_ipc()` or `pl$read_ipc()`, the argument `memory_map` is no longer used internally."

If you agree I can add that after the CI passes and merge directly

Reply:
No it is used. I have reverted the changes you made.
The following comment is generated by installing the latest commit at that point in time.
https://github.com/pola-rs/r-polars/pull/1104#discussion_r1615002017

The only problem is that there is probably no test at this time to confirm this behavior.

Reply:
Sorry if I miss something but this argument was removed here: https://github.com/pola-rs/r-polars/pull/1104/files#diff-02c4f6c445f96e42a1dcbf81cc271224cd2f115936c291f58990e2c0fc45ba66

So either it's a mistake and it should be put back or it's voluntary and it means the `memory_map` value passed by the user doesn't matter, right?

Reply:
No, it is not. The internal function `from_raw_ipc()` is `pl$read_ipc()` from the R side, but it is actually a separate function and is only called when reading R raw vectors.
Since raw vectors are not Arrow files, this argument has no meaning and is just removed.

Reply:
Look at the entire function. It is not removed.

Reply:
Ok, thanks for the explanation. There's still a failure in the user guide vignette that I'm trying to fix